modules/data-entry/src/main/frontend/src/questionnaireEditor/EditDialog.jsx

```diff
@@ -170,6 +172,7 @@ let EditDialog = (props) => {
           { dialogTitle() }
           </DialogTitle>
           <DialogContent>
+            { error && <Typography color="error">error</Typography>}
```

Comment:
Did you mean this instead?

```suggestion
            { error && <Typography color="error">{error}</Typography>}
```